environment.yml

```diff
@@ -2,9 +2,8 @@ name: ogusa-dev
 channels:
 - conda-forge
 dependencies:
-- python>=3.7.7, <3.11
+- python>=3.7.7, <=3.11
```

Comment:
@jdebacker. Similar to the recent changes in [PR #879]() in OG-Core, you should remove the "`, <=3.11`" from this line 5 in `environment.yml` and from line 22 of `setup.py`.